app/src/main/res/values/strings.xml

```diff
@@ -569,9 +569,10 @@
 
     <!--  FAQs  -->
     <string name="faqSearch">Search FAQ</string>
-  <string-array name="faq_question_answers">
+    <string name="welcome_excel_message">Convert your excel files into your most desired PDFs at ease</string>
```

Comment:
This string is related to welcome screen. 

Can you move it under this commment?
https://github.com/Swati4star/Images-to-PDF/blob/b775ca90cf672dac319d7116106f25db5ee2365d/app/src/main/res/values/strings.xml#L497

Reply:
Okay, I will do.